src/app/Features/CheckInconsistencies/LoopsAndMaxBlocks/index.tsx

```diff
@@ -125,28 +132,28 @@ export class CheckLoopsOnFlow extends BaseFeature {
 
     if (loopBlocksList.length > 0) {
       return [...loopBlocksList, loopBlocksList[0]].map(
-        (c) => `${flow[c]?.$title}`,
+        (c) => `${flow[c]?.$title}`
```

Comment:
maybe in this point the c variable can be substitute by other with a name more significant